src/components/ResourceList.tsx

```diff
@@ -16,15 +16,107 @@ import {
   ListPageBody,
   ListPageFilter,
 } from '@openshift-console/dynamic-plugin-sdk';
+import {
+  CheckCircleIcon,
+  ExclamationTriangleIcon,
+  LayerGroupIcon,
+  UploadIcon,
+  OutlinedHourglassIcon,
+} from '@patternfly/react-icons';
 import DropdownWithKebab from './DropdownWithKebab';
 
-const statusConditionsAsString = (obj: any) => {
-  if (!obj.status || !obj.status.conditions) {
-    return '';
+const getStatusLabel = (obj: any) => {
+  // Gateway or HTTPRoute
+  if (obj.kind === 'Gateway' || obj.kind === 'HTTPRoute') {
+    const acceptedCondition = obj.status?.conditions?.find((cond: any) => cond.type === 'Accepted' && cond.status === 'True');
+    const programmedCondition = obj.status?.conditions?.find((cond: any) => cond.type === 'Programmed' && cond.status === 'True');
+    const conflictedCondition = obj.status?.conditions?.find((cond: any) => cond.type === 'Conflicted' && cond.status === 'False');
```

Comment:
```suggestion
    const conflictedCondition = obj.status?.conditions?.find((cond: any) => cond.type === 'Conflicted' && cond.status === 'True');
```
?

Reply:
is the reason conflicted or the type conflicted?

Reply:
🤦